__tests__/server.test.js

```diff
@@ -293,15 +293,17 @@ describe('GET', () => {
                     .send({inc_votes: newVotes})
                     .expect(200)
                     .then(({body}) => {
-                        expect(body[0]).toHaveProperty('author', expect.any(String))
-                        expect(body[0]).toHaveProperty('title', expect.any(String))
-                        expect(body[0]).toHaveProperty('article_id', expect.any(Number))
-                        expect(body[0]).toHaveProperty('topic', expect.any(String))
-                        expect(body[0]).toHaveProperty('created_at', expect.any(String))
-                        expect(body[0]).toHaveProperty('votes', expect.any(Number))
-                        expect(body[0]).toHaveProperty('article_img_url', expect.any(String))
-                        expect(body[0]).toHaveProperty('votes', expect.any(Number))
-                        expect(body[0].votes).toBe(5)
+                        const {article} = body
+                        expect(article).toHaveProperty('author', expect.any(String))
+                        expect(article).toHaveProperty('title', expect.any(String))
+                        expect(article).toHaveProperty('article_id', expect.any(Number))
+                        expect(article.article_id).toBe(4)
+                        expect(article).toHaveProperty('topic', expect.any(String))
+                        expect(article).toHaveProperty('created_at', expect.any(String))
+                        expect(article).toHaveProperty('votes', expect.any(Number))
+                        expect(article).toHaveProperty('article_img_url', expect.any(String))
+                        expect(article).toHaveProperty('votes', expect.any(Number))
+                        expect(article.votes).toBe(5)
```

Comment:
Interesting that you assered specific info here but not for any of the other fields. Please test that the exact information we expect is coming back for all fields in line with article 2